### ISSUE TITLE
Better customisation of chunk loader blocks

### ISSUE THREAD

**Issue by leelawd** [ModdedMinecraft/BetterChunkLoader#2]
Dynamic loading of different chunk loader blocks instead of just basic iron and diamond (AKA multiple types of blocks per online / offline loader type)

**Comment by MindThoth** [comment]
Yeah that would be awesome because when I saw the config I was sure that was already possible so I wrote this : 

    Diamond {
        BlockType="minecraft:diamond_block"
        DefaultAlwaysOn=0
        Expiry=24
        MaxAlwaysOn=50
    }
    Gold {
        BlockType="minecraft:gold_block"
        DefaultAlwaysOn=0
        Expiry=12
        MaxAlwaysOn=50
    }
    Iron {
        BlockType="minecraft:iron_block"
        DefaultAlwaysOn=0
        Expiry=4
        MaxAlwaysOn=50
    }

Therefore would it be possible for a player to have like for example : 1 Diamond Chunk, 3 Gold Chunk and 9 Iron Chunk.

**Comment by leelawd** [comment]
The way the config is currently set up is completely static using @ConfigSerializable annotation (Which i didn't change when I was converting over other stuff from the previous code)

So I will have to completely re-do the configuration system in a similar fashion to my other plugins where i can easily allow things to be dynamically added and removed in a way that i know how to do (Unless @ConfigSerializable has some way to do this? Ill need to check.) I honestly don't know a whole lot about that annotation yet.
